File: ApiSuvesaPos/Negocio/Modelo/cxc/ventas/agente_ventas.vb
Namespace Modelo
    Public Class agente_ventas
        Public Property id As Integer
        Public Property cedula As String
        Public Property nombre As String
        Public Property telefono As String
        Public Property direccion As String
        Public Property correo As String
        Public Property activo As Boolean
        Public Property anulado As Boolean
        Public Property TipoPrecio As Integer
        Public Property PrecioDiferenciado As List(Of Modelo.preciodiferenciado)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/ventas/agente_ventas.cs
using System.Collections.Generic;

namespace Modelo
{
    public class agente_ventas
    {
        public int id { get; set; }
        public string cedula { get; set; }
        public string nombre { get; set; }
        public string telefono { get; set; }
        public string direccion { get; set; }
        public string correo { get; set; }
        public bool activo { get; set; }
        public bool anulado { get; set; }
        public int TipoPrecio { get; set; }
        public List<Modelo.preciodiferenciado> PrecioDiferenciado { get; set; }
    }
}
